modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/resolver/ParametersResolver.java

```diff
@@ -385,10 +383,8 @@ public void checkParameterGroupExclusiveness(ParameterizedModel model, Set<Strin
       resolver = getCollectionResolver((Collection) value);
     } else if (value instanceof Map) {
       resolver = getMapResolver((Map<Object, Object>) value);
-    } else if (isParameterResolver(modelProperties)) {
-      resolver = new StaticValueResolver<>(new StaticParameterResolver<>(value));
-    } else if (isTypedValue(modelProperties)) {
-      resolver = new StaticValueResolver<>(new TypedValue<>(value, DataType.fromObject(value)));
+    } else if (isWrapperType(modelProperties)) {
+      resolver = getStaticWrapperValueResolver(value, getWrapperModelProperty(modelProperties).get());
```

Comment:
why the word `static`? How do you know the wrapper type is static and why do you even care?

Reply:
`Static` because is a method that generates `ValueResolver` based on a "Static" value, the object "value" per se.
If the parameter originally was wrapped by one of the considered "WrapperTypes" I will ask to generate a Stack of `ValueResolver`, but with the first level of resolvers as a "Static Value Resolver". For example:
```
@Parameter
ParameterResolver<TypedValue<ParameterResolver<TypedValue<String>>>> nestedValue;
```

And the XML is the following:
```
<ext:some-op nestedValue="An static String"/>
```

I will generate the following Structure of ValueResolvers:
```
WrapperValueResolver(ParameterResolver) 
|-> WrapperValueResolver(TypedValue)
    |-> WrapperValueResolver(ParameterResolver)
        |-> StaticValueResolver(TypedValue)
```